docs/src/main/paradox/scala/http/client-side/host-level.md

```diff
@@ -56,13 +57,20 @@ the same pooled connections.
 <a id="using-a-host-connection-pool"></a>
 ## Using a Host Connection Pool
 
-The "pool client flow" returned by `Http().cachedHostConnectionPool(...)` has the following type:
+The "pool client flow" returned by @scala[`Http().cachedHostConnectionPool(...)`]@java[`Http.get(system).cachedHostConnectionPool(...)`] has the following type:
 
+@@@ div { .group-scala }
 ```scala
 Flow[(HttpRequest, T), (Try[HttpResponse], T), HostConnectionPool]
 ```
+@@@
+@@@ div { .group-java }
+```java
+Flow<Pair<HttpRequest, T>, Pair<Try<HttpResponse>, T>, HostConnectionPool>
+```
+@@@
 
-This means it consumes tuples of type `(HttpRequest, T)` and produces tuples of type `(Try[HttpResponse], T)`
+This means it consumes tuples of type @scala[`(HttpRequest, T)`]@java[`Pair<HttpRequest, T>`] and produces tuples of type @scala[`(Try[HttpResponse], T)`]@java[`Pair<Try<HttpResponse>, T>`]
```

Comment:
I'll replace with pairs, as 2 people think the same way, and the same thought crossed my mind